### PR TITLE
Add new fields for custom registry domains

### PR DESCRIPTION
**Pull request by paigecalvert**
SC: https://app.shortcut.com/replicated/story/63070/updates-to-custom-registry-domain-feature-for-ga

Preview: https://deploy-preview-908--replicated-docs.netlify.app/reference/custom-resource-application#proxyregistrydomain

### CODE REVIEW

**Comment by jonquil2002** [comment]
@ajp-io  This should have gone out Monday with the other custom registry domain info. Please check the changes I made to the links. I think everything else is correct.

File: docs/reference/custom-resource-application.md [hunk 1]
```diff
@@ -382,6 +384,44 @@ spec:
...
+  </tr>
+</table>
+
+## proxyRegistryDomain
+<table>
+  <tr>
+    <th>Description</th>
+    <td>
+      <p>The custom domain used for proxy.replicated.com. For more information, see <a href="/vendor/custom-domains#registry">Custom Registry Domains</a>.</p>
+      <p>Introduced in app manager v1.92.0.</p>
```
Comment (ajp-io) [inline docs/reference/custom-resource-application.md:412]: https://docs.replicated.com/release-notes/rn-app-manager#1912
```suggestion
      <p>Introduced in app manager v1.91.1.</p>
```

File: docs/reference/custom-resource-application.md [hunk 2]
```diff
@@ -382,6 +384,44 @@ spec:
   </tr>    
 </table>
 
+## replicatedRegistryDomain
+<table>
+  <tr>
+    <th>Description</th>
+    <td>
+      <p>The custom domain used for registry.replicated.com. For more information, see <a href="/vendor/custom-domains#registry">Custom Registry Domains</a>.</p>
+      <p>Introduced in app manager v1.92.0.</p>
```
Comment (ajp-io) [inline docs/reference/custom-resource-application.md:393]: This is the correct version, but there's actually a bug that prevents it from working. So we'll need to bump this version to our next KOTS version once it's out.

Comment (ajp-io) [reply docs/reference/custom-resource-application.md:393]: So maybe we should hold this until our next KOTS release and update it then.

**Review by jonquil2002 — APPROVED** [review]
lgtm 

@paigecalvert  I *think* we can leave the registry partial file in there for convenience because it doesn't publish. That way when the bug is fixed, we can just import the partial to the reference topic easily. But, lmk if we should delete that partial until we're ready for it.

**Comment by paigecalvert** [comment]
@jonquil2002 works for me!

**Comment by paigecalvert** [comment]
@jonquil2002 looks good to merge (I realize the broken link is just bc this branch doesn't have the new custom domains topic). I can't click approve since technically I opened the PR :)

**Pull request merged**